core/ui/SideBar/Open.tid

```diff
@@ -10,22 +10,31 @@ caption: {{$:/language/SideBar/Open/Caption}}
 \end
 
 <$list filter="[list[$:/StoryList]]" history="$:/HistoryList" storyview="pop">
-<div style="position: relative;">
-<$droppable actions=<<drop-actions>>>
-<div class="tc-droppable-placeholder">
-&nbsp;
-</div>
-<div>
-<$button message="tm-close-tiddler" tooltip={{$:/language/Buttons/Close/Hint}} aria-label={{$:/language/Buttons/Close/Caption}} class="tc-btn-invisible tc-btn-mini">&times;</$button> <$link to={{!!title}}><$view field="title"/></$link>
-</div>
-</$droppable>
-</div>
+  <div style="position: relative;">
```

Comment:
It's unclear what changes have been made to this file (if any) because of the addition of indenting. It would be clearer to either not change it, or change the indenting in a separate PR that can be merged more quickly.

Reply:
There should be indentation only. ... I'll have to check this. I didn't add the style thing intentionally.

Reply:
It's already there at tiddlywiki.com see: https://tiddlywiki.com/#%24%3A%2Fcore%2Fui%2FSideBar%2FOpen ... May be a problem with github highlighting.